src/content/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings.mdx

```diff
@@ -7,68 +7,39 @@ tags:
 metaDescription: How to access and edit New Relic account settings.
 ---
 
-To view and edit account-related settings, go to [one.newrelic.com](https://one.newrelic.com) and in the top right corner, select the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown). 
+To view and edit account-related and user-management-related settings, go to [one.newrelic.com](https://one.newrelic.com) and in the top right corner, select the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown). 
 
 Keep reading for tips on how to find different kinds of account settings in the UI. 
 
-## Pricing and billing settings [#pricing]
+## Pricing, billing, and usage UI [#pricing]
 
-To view and edit some billing-related settings, you must be a [full user](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type). Basic users have some restrictions.
+Note that [user permissions](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/) may impact your access to some account settings. 
 
-How you access billing-related UI depends on which [pricing plan](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#pricing-plans) your New Relic account is on: 
+To find billing-related UI, go to [one.newrelic.com](https://one.newrelic.com) and, in the upper right corner, click the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown) to access various account settings. Here are the billing-related UI pages available there: 
```

Comment:
"Here are the billing-related UI pages available there:" I prefer something more like:

"Here's what you can do there:"